src/test/scala/org/constellation/SnapshotHashIntegrityTest.scala

```diff
@@ -27,9 +27,9 @@ class SnapshotHashIntegrityTest extends AnyFreeSpec with Matchers with BeforeAnd
 
   "snapshot info read" in {
     val storage = SnapshotInfoLocalStorage[IO]("src/test/resources")
-    val snapshotInfo = storage.read("snapshot-info").rethrowT.unsafeRunSync()
+    val snapshotInfo = storage.read("snapshot-info-v2").rethrowT.unsafeRunSync()
```

Comment:
Is that the old snapshot? If yes why did the hash change?

Reply:
No, just randomly selected snapshot from the testnet, to test the storage service